### PR TITLE
Spanish translation of 2019-06-09-ltb-pieter-wuille-jonas-nick

### PR DESCRIPTION
**Pull request by aarmoa**
Added a new file with the translation from 2019-06-09-ltb-pieter-wuille-jonas-nick

### CODE REVIEW

File: lets-talk-bitcoin-podcast/2019-06-09-ltb-pieter-wuille-jonas-nick.es.md [hunk 1]
```diff
@@ -0,0 +1,279 @@
...
+
+Stephanie: ¿Schnorr cambió de idea al respecto de la patente?
+
+Pieter: Hasta donde yo sé él siempre sostuvo que DSA en realidad infringía su patente.
+
+Jonathan: Esta siempre fue una pregunta que me hice. Escucho que Bitcoin es descentralizado y siempre intento entender lo que eso implica y lo que no. Uno escuchó hablar sobre quién tiene el control de Bitcoin y quién está realmente a cargo y es responsable por el proyecto. Escuchamos todas las protecciones que los contribuyentes a Bitcoin Core toman respecto de la libre opinión por contribuir al repositorio pero aclarando que no son los actores comerciales que participan en las actividades, que es por lo que existen fuertes protecciones para las contribuciones de código abierto
+<sup id="a6">[6](#f6)</sup>. ¿Por qué algo como una patente supondría un freno para que una tecnología superior sea implementada en un repositorio de un proyecto descentralizado y de código abierto? No entiendo dónde está el factor limitante. ¿Fue sólo debido a que las bibliotecas no eran estándares como mencionaste antes?
```
Comment (adamjonas) [inline lets-talk-bitcoin-podcast/2019-06-09-ltb-pieter-wuille-jonas-nick.es.md:106]:
```suggestion
Jonathan: Esta siempre fue una pregunta que me hice. Escucho que Bitcoin es descentralizado y siempre intento entender lo que eso implica y lo que no. Uno escuchó hablar sobre quién tiene el control de Bitcoin y quién está realmente a cargo y es responsable por el proyecto. Escuchamos todas las protecciones que los contribuyentes a Bitcoin Core toman respecto de la libre opinión por contribuir al repositorio pero aclarando que no son los actores comerciales que participan en las actividades, que es por lo que existen fuertes protecciones para las contribuciones de código abierto<sup id="a6">[6](#f6)</sup>. ¿Por qué algo como una patente supondría un freno para que una tecnología superior sea implementada en un repositorio de un proyecto descentralizado y de código abierto? No entiendo dónde está el factor limitante. ¿Fue sólo debido a que las bibliotecas no eran estándares como mencionaste antes?
```

File: lets-talk-bitcoin-podcast/2019-06-09-ltb-pieter-wuille-jonas-nick.es.md [hunk 2]
```diff
@@ -0,0 +1,279 @@
...
+<br/>
+<br/>
+<br/>
+<br/>
+<br/>
+<hr/>
+<a id="f1">[1]</a> "Let's Talk Bitcoin" [↩](#a1)<br/>
```
Comment (adamjonas) [inline lets-talk-bitcoin-podcast/2019-06-09-ltb-pieter-wuille-jonas-nick.es.md:263]: this footnote isn't working

**Review by adamjonas — COMMENTED** [review]
Sorry for the late reply and thanks for doing this!

You need to add an _index.es.md file in accordance with https://github.com/bitcointranscripts/bitcointranscripts#i18n for the index page to show up.

Also the footnotes seem like a good idea, but they don't render all that well and I think an inline () would be more effective rather than having readers click around.

Other than that, it looks like a great addition!

**Comment by aarmoa** [comment]
@adamjonas Sorry for the late reply. I somehow missed the notification.
Do you think we should change all footnotes and insert them right inside the text in between parentheses?

**Comment by kouloumos** [comment]
Thanks for the update @aarmoa! Parentheses sounds good and I think that's what Jonas meant with 

>  I think an inline () would be more effective rather than having readers click around.

**Comment by aarmoa** [comment]
I have made all requested changes

**Pull request merged**